pkg/operator/tektonchains.go

```diff
@@ -29,9 +39,16 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+const(
```

Comment:
I'm sorry... I didn't realize that you are not using a static string but a result of a function. Please revert, this doesn't provide any value